app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def missionsystem_main
+  end
+
+  def missionsystem_help
+  end
+
+  def missionsystem_about
+  end
+end
```

Comment:
The static pages are handled by `high_voltage ` gem. Please learn how to use this gem.